crypto/s2n_hash.c

```diff
@@ -205,26 +211,26 @@ static int s2n_evp_hash_init(struct s2n_hash_state *state, s2n_hash_algorithm al
         r = 1;
         break;
     case S2N_HASH_MD5:
-        r = EVP_DigestInit_ex(state->digest.evp.primary.ctx, EVP_md5(), NULL);
+        r = EVP_DigestInit_ex(state->digest.high_level.evp.ctx, EVP_md5(), NULL);
```

Comment:
I agree, good suggestion. I've done this for all high-level implementation s2n_hash calls where there is a common EVP_MD_CTX in each case of the switch statement.